StratisCore.UI/src/app/wallet/transactions/transactions.component.ts

```diff
@@ -60,9 +59,12 @@ export class TransactionsComponent implements OnInit, OnDestroy {
   public ngOnInit(): void {
     this.transactions = this.walletService.walletHistory()
       .pipe(map((historyItems => {
-        return (null != historyItems && historyItems.length > 0)
-          ? TransactionInfo.mapFromTransactionsHistoryItems(historyItems, this.maxTransactionCount)
-          : [];
+        return ((null != historyItems && historyItems.length > 0))
+        ? this.stakingOnly 
+          ?  TransactionInfo.mapFromTransactionsHistoryItems(historyItems.filter(items => items.type === "staked"), this.maxTransactionCount)
```

Comment:
We can't do this, It needs to be it's own `Subject` keyed by filter. I will also make changes to `getHistory` on full node to support transaction types

Reply:
Ok, so we need an updated submodule to run it. Leave as is for now and adjust when we have fn ready for it?